presto-raptor/pom.xml

```diff
@@ -191,7 +192,6 @@
         <dependency>
             <groupId>com.facebook.presto</groupId>
             <artifactId>presto-main</artifactId>
```

Comment:
Move this out of the test section. Is @electrum ok with making Raptor an internal plugin?